Solutions/CiscoSEG/Analytic Rules/CiscoSEGUnclassifiedLink.yaml

```diff
@@ -0,0 +1,32 @@
+id: 9cb4a02d-3708-42ba-b33b-0fdd360ce4b6
+name: Cisco SEG - Unexpected link
+description: |
+  'Detects mails with suspicious links.'

```

Comment:
Can we give more details in description why the link is suspicious as there is one more analytic rule with same description.

Reply:
changed description